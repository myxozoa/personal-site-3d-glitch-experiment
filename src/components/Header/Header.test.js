import React from 'react';
import { shallow } from 'enzyme';

import Component from './index';

describe('<Header />', () => {
  it('renders correctly', () => {
    const component = shallow(<Component />);

    expect(component).toMatchSnapshot();
  });
});