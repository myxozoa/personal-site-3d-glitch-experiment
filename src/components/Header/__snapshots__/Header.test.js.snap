// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<Fragment>
  <div
    className="normal closeButton button"
    onClick={[Function]}
  >
    <div />
    <div />
    <div />
  </div>
  <div
    className="show"
    onClick={[Function]}
  >
    <h1
      className="name"
    >
      <div
        className="glitch"
        data-text="ronnie"
        style={
          Object {
            "--offset": "1.5s",
          }
        }
      >
        ronnie
      </div>
      <div
        className="lastName"
      >
        <span
          className="glitch"
          data-text="miksch"
          style={
            Object {
              "--offset": "1.5s",
            }
          }
        >
          miksch
        </span>
        <div
          className="periodContainer"
        >
          <div
            className="period glitch"
            data-text="."
            style={
              Object {
                "--offset": "1.5s",
              }
            }
          />
        </div>
      </div>
    </h1>
    <Navigation
      show={true}
    />
  </div>
</Fragment>
`;
